backend/requirements.txt

```diff
@@ -0,0 +1,59 @@
+aiohttp==3.9.1
```

Comment:
Tried running the commands on my local machine with a Virtual Enviornment and had less dependencies in my requirements.txt file (using pip install -r requirements.txt failed). When I did a pip freeze these were the requirements I got were in the attached requirements file. 
[requirements.txt](https://github.com/user-attachments/files/17020836/requirements.txt)